calico_node/startup/startup.go

```diff
@@ -84,14 +84,17 @@ func main() {
 	node := getNode(client, nodeName)
 
 	// Configure and verify the node IP addresses and subnets.
-	configureIPsAndSubnets(node)
+	checkConflicts := configureIPsAndSubnets(node)
+
+	// If we report an IP change (v4 or v6) we should verify there are no
+	// conflicts between Nodes.
+	if checkConflicts && os.Getenv("DISABLE_NODE_IP_CHECK") != "true" {
```

Comment:
What do we respect for other true/false variables?  Seems fine/good to do, but I'm happy to merge without.